docs/setup/getting-started.md

```
@@ -67,32 +43,50 @@ pnpm install
+1. Prepare Docker Compose environment variables used for local databases:
+
+Ensure `docker/.env` exists and has values. If the file doesn't exist, create it with at least:
+
+```env
+# docker/.env
+NEO4J_USER=neo4j
+NEO4J_PASSWORD=letmein
+```
 
-1. Configure GitHub App:
+2. Create `.env.local` for app secrets used by Next.js (if applicable):
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)